src/Tasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -2489,20 +2490,14 @@ Copyright (C) Microsoft Corporation. All rights reserved.
     ====================================================================================================
   -->
   <Target Name="GenerateBindingRedirectsUpdateAppConfig"
-    AfterTargets="GenerateBindingRedirects"
-    Condition="'$(AutoGenerateBindingRedirects)' == 'true' and '$(GenerateBindingRedirectsOutputType)' == 'true' and Exists('$(_GenerateBindingRedirectsIntermediateAppConfig)')">
-
-    <PropertyGroup>
-      <AppConfig>$(_GenerateBindingRedirectsIntermediateAppConfig)</AppConfig>
```

Comment:
I'm not sure removing this is safe--I can imagine a custom target that uses `$(AppConfig)` and wants the updated one. Does it need to change?

Reply:
I don't recall changing this.  I can revert.

Reply:
I think it was a simplification of the exists check.  I will revert.